src/test/java/com/rtb/projectmanagementtool/project/ProjectDataTest.java

```diff
@@ -0,0 +1,193 @@
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.gson.Gson;
+import com.rtb.projectmanagementtool.project.ProjectData;
+import com.rtb.projectmanagementtool.project.UserProjectRole;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProjectDataTest {
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  // Projects
+  private final String PROJECT1_NAME = "Project 1";
+  private final String PROJECT1_DESC = "Description of Project 1";
+
+  private final String PROJECT2_NAME = "Project 2";
+  private final String PROJECT2_DESC = "Description of Project 2";
+
+  private final String PROJECT3_NAME = "Project 3";
+  private final String PROJECT3_DESC = "Description of Project 3";
+
+  private final String PROJECT4_NAME = "Project 4";
+  private final String PROJECT4_DESC = "Description of Project 4";
+
+  // Project Creators
+  private final Long USER1 = 1l;
+  private final Long USER2 = 2l;
+  private final Long USER3 = 3l;
+  private final Long USER4 = 4l;
+
+  // Project Tasks
+  private final Long TASK1 = 1l;
+  private final Long TASK2 = 2l;
+  private final Long TASK3 = 3l;
+  private final Long TASK4 = 4l;
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Test project creation given name, desc, and creator
+  @Test
+  public void testCreateProject() {
```

Comment:
nit:  I don't think we need to prefix all test methods with `test` since the @Test annotation already does that for us.